### DISCUSSION TITLE
'left,right,top,bottom' has been transformed to 'inset', how can I disable it?

### DISCUSSION THREAD

**Discussion by ShookLyngs** [postcss/postcss#1719] [Q&A]
`inset` property is not supported in `Chrome < 87` environment, so I wanted to disabled the transformation.
I tried to set the browserlist config to disable the transformation but it did not work:
```
[
  "chrome 86", 
  "not chrome > 86"
]
```

Does anyone have experice with this issue, could really need some help here.

**Comment by ai** [comment]
I can’t see `inset` support in `postcss-preset-env` or Autoprefixer.

Who add it in your CSS build system?

**Reply by ShookLyngs** [reply]
> I can’t see `inset` support in `postcss-preset-env` or Autoprefixer.
> 
> Who add it in your CSS build system?

Thanks for the tip, I have solved the issue, turns out it was `esbuild` in `vite` actually transformed `left, right, top, bottom` to `inset`.
